### PR TITLE
Update readme.txt #23

### PR DESCRIPTION
**Pull request by mindctrl**
#23

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,33 +1,36 @@
 === Restrict Content ===
 Author URI: http://pippinsplugins.com
 Author: Pippin Williamson
-Contributors: mordauk
+Contributors: mordauk, mindctrl, nosegraze
 Donate link: http://pippinsplugins.com/support-the-site
-Tags: Restrict content, member only, registered, logged in, restricted access, restrict access, limiit access, read-only, read only
+Tags: Restrict content, member, members, membership, memberships, member only, registered, registration form, login form, logged in, restricted access, restrict access, limiit access, read-only, read only, access control
```
Comment (pippinsplugins) [inline readme.txt]: Too many tags. 12 is the max.

File: readme.txt [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
+Restrict Content to registered users only. This is a simple membership plugin that will allow you to easily restrict complete posts / pages to logged in users only. Levels of restriction may also be set. For example, you can restrict content to only Administrators, Editors, Authors, Contributors, and Subscribers.
 
 == Description ==
 
-Restrict Content to registered users only. This is a simple plugin that will allow you to easily restrict complete posts / pages to logged in users only. Levels of restriction may also be set. 
-For example, you can restrict content to only Administrators, Editors, Authors, and Subscribers.
+Restrict Content to registered users only. This is a simple membership plugin that will allow you to easily restrict complete posts / pages to logged in users only. Levels of restriction may also be set.
+For example, you can restrict content to only Administrators, Editors, Authors, Contributors, and Subscribers.
```
Comment (pippinsplugins) [inline readme.txt:16]: Let's add the actual shortcodes here.

**Review by pippinsplugins — CHANGES_REQUESTED** [review]

File: readme.txt [hunk 3]
```diff
@@ -3,7 +3,7 @@ Author URI: http://pippinsplugins.com
 Author: Pippin Williamson
 Contributors: mordauk, mindctrl, nosegraze
 Donate link: http://pippinsplugins.com/support-the-site
-Tags: Restrict content, member, members, membership, memberships, member only, registered, registration form, login form, logged in, restricted access, restrict access, limiit access, read-only, read only, access control
+Tags: Restrict content, member, members, membership, memberships, member only, registration form, login form, restricted access, limit access, read only, access control
```
Comment (pippinsplugins) [inline readme.txt]: There are 13 here still. 12 is max per .org guidelines.

Comment (mindctrl) [reply readme.txt]: @pippinsplugins I count 12.

Comment (pippinsplugins) [reply readme.txt]: @mindctrl I count 13 when I include the imaginary one in my head

**Review by pippinsplugins — APPROVED** [review]

**Pull request merged**